components/hab/src/command/studio/docker.rs

```diff
@@ -64,12 +64,24 @@ pub fn start_docker_studio(_ui: &mut UI, args: &[OsString]) -> Result<()> {
     } else {
         ""
     };
+
+    let local_cache_key_path = default_cache_key_path(None);
+    if !local_cache_key_path.exists() {
+        return Err(Error::FileNotFound(
```

Comment:
Re-using this error variant is fine with me, but I think we should include *where* we looked for the path to exist. This error could be confusing in the event of an env var override.

Reply:
Isn't `local_cache_key_path` where we looked?

Reply:
Fair point. I'm not sure if I just missed that or was thinking about something else. This looks good to me.